PurchasesTests/Networking/HTTPClientTests.swift

```diff
@@ -308,7 +308,7 @@ class HTTPClientTests: XCTestCase {
         let path = "/a_random_path"
         var headerPresent = false
         
-        let version = "" // for unit tests the version will empty
+        let version = RCSystemInfo.appVersion()
```

Comment:
In case the description isn't clear enough (it probably isn't), if instead of doing this, on this line I did
`let version = Bundle.main.infoDictionary?["CFBundleShortVersionString"]`, that actually returns `""` and the test fails. 
Calling `RCSystemInfo.appVersion()` returns `13.0`, but after all, what we really want is to make sure that it sends the value obtained in `RCSystemInfo.appVersion()`, so this test should work well as a replacement.

Reply:
Makes sense. 🛳 it